include/RAJA/exec-agency/forallN_agency.hxx

```diff
@@ -0,0 +1,211 @@
+/*!
```

Comment:
This file doesn't work, and actually doesn't get included right now; further development will happen in another branch (this will probably be more or less entirely rewritten)